### PR TITLE
in-floor valve and blinds support

### PR DESCRIPTION
**Pull request by thepiwo**
depends on: https://github.com/inverse/python-iolite-client/pull/220

### CODE REVIEW

File: tests/test_config_flow.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ async def test_flow_invalid_scan_interval_range(
         DOMAIN, context={"source": SOURCE_USER}
     )
 
-    with pytest.raises(voluptuous.error.MultipleInvalid):
+    with pytest.raises(data_entry_flow.InvalidData):
```
Comment (inverse) [inline tests/test_config_flow.py:32]:
```suggestion
    with pytest.raises((voluptuous.error.MultipleInvalid):
```

Shouldnt this be the error asserted as the value should be at least 15?

Comment (thepiwo) [reply tests/test_config_flow.py]: sadly no, I tried this but the validation seems to have changed https://github.com/inverse/home-assistant-iolite-component/actions/runs/8663239816/job/23756953149?pr=31#step:5:214

maybe I'll just rollback the dependency update

**Comment by inverse** [comment]
Interesting when I was testing this locally the other day it was working - let me have another think :thinking:

**Comment by thepiwo** [comment]
@inverse thank you for taking the time, I'm not super familiar with python best practices here, in other langues I would publish a lockfile to the repo

**Comment by inverse** [comment]
Yeah i'm more used to using poetry which includes the lock file of all the indirect dependencies too. Perhaps we could migrate to that?

**Comment by thepiwo** [comment]
@inverse yeah we could consider that, for the current PR the tests do pass now, do we want to merge as is?

**Review by inverse — APPROVED** [review]
Go for it - yeah it could definitely be a future improvement

**Pull request merged**